soliket/tests/test_mflike.py

```diff
@@ -73,26 +73,13 @@
 
 
 class MFLikeTest(unittest.TestCase):
-    orig = False
 
     def setUp(self):
         from cobaya.install import install
 
-        install({"likelihood": {"mflike.MFLike": None}},
-                path=packages_path, skip_global=False, force=True, debug=True)
         install({"likelihood": {"soliket.MFLike": None}},
                 path=packages_path, skip_global=False, force=True, debug=True)
 
-    def get_mflike_type(self, as_string=False):
-        if self.orig:
-            t = "mflike.MFLike"
-        else:
-            t = "soliket.MFLike"
-
-        if as_string:
-            return t
-        else:
-            return eval(t)
 
     @pytest.mark.skip(reason="don't want to install 300Mb of data!")
```

Comment:
Is there a way to write these tests such that we indeed do not need to install the full cache of data from the installable likelihood? We need to stop skipping the tests, and the best way to do this would be not to have to download the large file anymore.